### ISSUE TITLE
Multiple parking areas close to eachother won't work

### ISSUE THREAD

**Issue by dutchplayers** [kasuganosoras/esx_realparking#10]
Hello!

Thanks for the great resource!

I noticed you can't change the head of the parking lot. So I thought it might work to add every spot in a parking lot, but it fails. For example, once I have created 10 small parking spaces next to each other, the resource will no longer load the vehicles properly.

It would be nice if you can change the heading or shape of the real parking or if it is possible to put several small parking spaces next to each other.

**Comment by AquariusGhost** [comment]
i dont believe your supposed to create each space individually, your actually supposed to just create a big zone by inputting the center location as the parking zone and then adjust the size to fit
